mvcc/backend/backend.go

```diff
@@ -163,6 +165,20 @@ func (b *backend) ForceCommit() {
 }
 
 func (b *backend) Snapshot() Snapshot {
+	stopc, donec := make(chan struct{}), make(chan struct{})
+	go func() {
+		defer close(donec)
+		now := time.Now()
+		for {
+			select {
+			case <-time.After(snapshotWarningTimeout):
+				plog.Warningf("snapshotting is taking more than %v to finish [started at %v]", time.Since(now).Seconds(), now)
```

Comment:
`snapshotting is taking more than 4.014502321 to finish [started at 2017-05-04 15:18:56.157129795 -0700 PDT]`